rs/consensus/src/idkg/payload_builder.rs

```diff
@@ -617,7 +620,7 @@ pub(crate) fn create_data_payload_helper_2(
             .is_some_and(|(pid, _)| idkg_payload.available_pre_signatures.contains_key(pid))
         {
             *matched_pre_signatures_per_key_id
-                .entry(context.key_id())
+                .entry(context.key_id().try_into().unwrap())
```

Comment:
I guess instead of unwrapping we could also skip non-idkg contexts

Reply:
Should not make a difference, as `key_id` can only return the two IDKG variants, but this is code not owned by consensus, so I think, just to be on the safe side, lets skip.

Reply:
Implemented